app/components/post_draft/post_draft.tsx

```diff
@@ -73,6 +74,11 @@ function PostDraft({
         setCursorPosition(message.length);
     }, [channelId, rootId]);
 
+    useEffect(() => {
```

Comment:
What is the impact if you use `useLayoutEffect` instead?

Reply:
`useLayoutEffect` will run before painting into the screen. I would prefer if the measure is done after painting. But 0/5.

Reply:
Now that you mention it, I wanted to revisit this, you are right.
No, the post draft is not the last thing that renders in a channel. I thought it might be (and was convenient) but after some tests, it may not be.
Also, I completely forgot about the tablet scenario :facepalm: 
I will revisit this.